JS/Lesson34/goksel/recipe-app/src/modules/user/userSlice.tsx

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const userSlice = createSlice({
+    name: "user",
+    initialState: { isSignedIn: false }, 
+    reducers: {
+        toggleSignIn: (state) => {
```

Comment:
It's safer to have separate action types for sign in and sign out as in real life you don't just toggle it, often when user clicks log out button (or performs similar action) we want to be 100% sure that he/she signs out.

Reply:
Fixed